package.json

```diff
@@ -33,12 +33,13 @@
 		"start": "react-scripts start",
 		"build": "react-scripts build",
 		"test": "react-scripts test --passWithNoTests --watchAll=false",
+		"test-modules": "react-scripts test ModulesPage.slice.spec.ts --watchAll=false",
```

Comment:
I do want to have a command for running specific test cases if possible, I think being able to see which test cases failed in specific is a good thing. If you guys do not think it is necessary I can remove it with a quick commit.